### ISSUE TITLE
performance on imagenet ILSVRC2012

### ISSUE THREAD

**Issue by hengck23** [zeusees/Mnasnet-Pretrained-Model#2]
i evaluate your model 0.2642-imagenet-masnet-194-best.params on the standard validation split of imagenet ILSVRC2012. This is my results:

```
# valid_num = 50000
# valid_acc1 (err)= 0.705440 (0.294560)
# valid_acc5 (err)= 0.897400 (0.102600)
# valid_loss = 1.323303

```

i do not get  top1/5 accuracy (73.6% / 91.52%) as claimed?

**Comment by szad670401** [comment]
Can you show your test code?

**Comment by tzzcl** [comment]
Hi, I use your pretrained models for testing and I use gluon.DataLoader (not rec) for testing.

The testing results is:
<pre> 
('accuracy', 0.708) ('top_k_accuracy_5', 0.89872)
validation: err-top1=0.292000 err-top5=0.101280
</pre> 

Could you please explain the difference between rec and DataLoader and whether I make a mistake during training?
